### PR TITLE
Fix test_system_health.py script related to system-ready's publish_system_status function

### PR DESCRIPTION
**Pull request by sg893052**
Include valid input check for system status in test

#### Why I did it
Fix for issue#14871

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Include valid input check for system status in test

#### How to verify it
As said in PR, break the publish_system_status function code and build system-health package.
UT:
tests/test_system_health.py:741: in <module>
    ???
/usr/local/lib/python3.9/dist-packages/mock/mock.py:1423: in patched
    return func(*newargs, **newkeywargs)
tests/test_system_health.py:738: in test_publish_system_status
    assert call_args in expected_calls
E   AssertionError: assert call('FAIL') in [(('UP',), {}), (('DOWN',), {})]
ERROR tests/test_system_health.py - AssertionError: assert call('FAIL') in [(('UP',), {}), (('DOWN',), {})]

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

**Comment by sg893052** [comment]
This PR is to address https://github.com/sonic-net/sonic-buildimage/issues/14871

File: src/system-health/tests/test_system_health.py [hunk 1]
```diff
@@ -761,14 +761,20 @@ def test_post_system_status():
...
+
 @patch('health_checker.sysmonitor.Sysmonitor.print_console_message', MagicMock())
+@patch('health_checker.sysmonitor.Sysmonitor.post_system_status', MagicMock())
 def test_publish_system_status():
     sysmon = Sysmonitor()
     sysmon.publish_system_status('UP')
-    result = swsscommon.SonicV2Connector.get(MockConnector, 0, "SYSTEM_READY|SYSTEM_STATE", 'Status')
-    assert result == "UP"
+    sysmon.publish_system_status('DOWN')
+
```
Comment (DavidZagury) [inline src/system-health/tests/test_system_health.py:771]:
```suggestion
    sysmon.publish_system_status('UP')
    result = swsscommon.SonicV2Connector.get(MockConnector, 0, "SYSTEM_READY|SYSTEM_STATE", 'Status')
    assert result == 'UP'

    sysmon.publish_system_status('DOWN')
    result = swsscommon.SonicV2Connector.get(MockConnector, 0, "SYSTEM_READY|SYSTEM_STATE", 'Status')
    assert result == 'DOWN'
```
Would make it possible to make sure that the publish function has in fact worked, and the DB was updated as it should

Comment (sg893052) [reply src/system-health/tests/test_system_health.py:775]: Taken care

**Comment by prsunny** [comment]
Please use proper title description.

**Comment by adyeung** [comment]
@DavidZagury pls review and signoff

**Review by DavidZagury — APPROVED** [review]
Looks good

**Review by Praveen-Brcm — APPROVED** [review]

**Pull request merged**